ui/src/main/java/edu/wpi/grip/ui/MainWindowController.java

```diff
@@ -241,6 +244,29 @@ protected void quit() {
       SafeShutdown.exit(0);
     }
   }
+  
+  /**
+   * Controls the export button in the main menu. Opens a filechooser with language selection.
+   * The user can select the language to export to, save location and file name.
+   * @param actionEvent Unused event passed by the controller.
+   */
+  public void generate(ActionEvent actionEvent) {
+    final FileChooser fileChooser = new FileChooser();
+    fileChooser.setTitle("Export to");
+    fileChooser.getExtensionFilters().add(new ExtensionFilter(Language.JAVA.name, "*.java"));
+    fileChooser.getExtensionFilters().add(new ExtensionFilter(Language.CPP.name, "*.cpp"));
+    fileChooser.getExtensionFilters().add(new ExtensionFilter(Language.PYTHON.name, "*.py"));
+    fileChooser.setInitialFileName("Pipeline");
```

Comment:
We could but a user likely wouldn't want to put their code there. This way if a user exports their code and then decides to change the pipeline and export again the directory remains where they last set it.